### PR TITLE
Add section: Enable Java tests for unmanaged folders

### PR DESCRIPTION
**Pull request by jdneo**
Signed-off-by: Sheng Chen <sheche@microsoft.com>

### CODE REVIEW

**Comment by jdneo** [comment]
@nickzhums

File: docs/java/java-testing.md [hunk 1]
```diff
@@ -38,7 +38,17 @@ The [Test Runner for Java](https://marketplace.visualstudio.com/items?itemName=v
 
 ## Project Setup
 
-**Note**: If you have already setup your Java test framework in your project, you can skip to the [Features](#features) section.
+> **Note**: If you have already setup your Java test framework in your project, you can skip to the [Features](#features) section.
+
+### Enabling test framework for your unmanaged folder
```
Comment (nickzhums) [inline docs/java/java-testing.md:43]:
```suggestion
### Enabling testing and adding test framework JARs to your project
```

Comment (nickzhums) [reply docs/java/java-testing.md:43]: let's make it more explicit

Comment (jdneo) [reply docs/java/java-testing.md]: Addressed.

**Review by nickzhums — APPROVED** [review]

**Pull request merged**